src/GuestConfiguration/GuestConfiguration/GuestConfiguration.csproj

```diff
@@ -12,6 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.GuestConfiguration" Version="1.1.0" />
+    <PackageReference Include="Microsoft.Azure.Management.PolicyInsights" Version="2.0.2" />
```

Comment:
See above.  Only the PolicyInsights module may take a dependency on the PolicyInsights management client

Reply:
Updated.